content/docs/reference/config/installation-options-version-comparison/index.md

```diff
@@ -0,0 +1,445 @@
+| `gateways.istio-ingressgateway.ports.name` | `http2-grafana` | `https-kiali` | `` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15031` | `15030` | `` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `http2-grafana` | `https-prometheus` | `` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `http2-grafana` | `https-grafana` | `` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15031` | `15032` | `` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `http2-grafana` | `https-tracing` | `` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15031` | `15443` | `` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `http2-grafana` | `tls` | `` | `` |
+| `gateways.istio-ingressgateway.ports.targetPort` | `15031` | `15020` | `` | `` |
+| `gateways.istio-ingressgateway.ports.name` | `http2-grafana` | `status-port` | `` | `` |
+| `gateways.istio-ingressgateway.secretVolumes.secretName` | `istio-ingressgateway-ca-certs` | `istio-ingressgateway-certs` | `` | `` |
+| `gateways.istio-ingressgateway.secretVolumes.mountPath` | `/etc/istio/ingressgateway-ca-certs` | `/etc/istio/ingressgateway-certs` | `` | `` |
+| `gateways.istio-egressgateway.enabled` | `true` | `false` | `` | `` |
+| `gateways.istio-egressgateway.ports.name` | `https` | `http2` | `` | `` |
+| `gateways.istio-egressgateway.ports.name` | `https` | `tls` | `` | `` |
+| `gateways.istio-egressgateway.secretVolumes.secretName` | `istio-egressgateway-ca-certs` | `istio-egressgateway-certs` | `` | `` |
+| `gateways.istio-egressgateway.secretVolumes.mountPath` | `/etc/istio/egressgateway-ca-certs` | `/etc/istio/egressgateway-certs` | `` | `` |
+| `gateways.istio-ilbgateway.ports.name` | `tcp-dns` | `grpc-pilot-mtls` | `` | `` |
+| `gateways.istio-ilbgateway.ports.name` | `tcp-dns` | `grpc-pilot` | `` | `` |
+| `gateways.istio-ilbgateway.ports.name` | `tcp-dns` | `tcp-citadel-grpc-tls` | `` | `` |
+| `gateways.istio-ilbgateway.secretVolumes.secretName` | `istio-ilbgateway-ca-certs` | `istio-ilbgateway-certs` | `` | `` |
+| `gateways.istio-ilbgateway.secretVolumes.mountPath` | `/etc/istio/ilbgateway-ca-certs` | `/etc/istio/ilbgateway-certs` | `` | `` |
+
+## `certmanager` options
+
+| Key | Old Default Value | New Default Value | Old Description | New Description|
+| --- | --- | --- | --- | --- |
+| `certmanager.tag` | `v0.3.1` | `v0.6.2` | `` | `` |
+
+## `kiali` options
+
+| Key | Old Default Value | New Default Value | Old Description | New Description|
+| --- | --- | --- | --- | --- |
+| `kiali.tag` | `istio-release-1.0` | `v0.14` | `` | `` |
+
+## `pilot` options
+
+| Key | Old Default Value | New Default Value | Old Description | New Description|
+| --- | --- | --- | --- | --- |
+| `pilot.autoscaleMax` | `1` | `5` | `` | `` |
+| `pilot.traceSampling` | `100.0` | `1.0` | `` | `` |
+
+
+# New configuration options that have been added to the current release 
+
+## `istio_cni` options
```

Comment:
To make it easy for readers to know on what part of the document they are, each of this headings should provide that signal. In this section, all headings should indicate that the key/value pairs are new to the option in question.

```suggestion
## New `istio_cni` key/value pairs
```